docs/content/en/install/eks/_index.md

```diff
@@ -40,24 +40,138 @@ On this infrastructure the K8ssandra installation will consist of the following
+
+Terraform install example for Ubuntu Linux:
+
+```bash
+sudo apt-get update && sudo apt-get install -y gnupg software-properties-common curl
+curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+sudo apt-get update && sudo apt-get install terraform
+```
+
+Verify the installation:
+
+```bash
+terraform version
+```
+
+**Output**:
+
+```bash
+Terraform v1.0.0
+on darwin_amd64
+
+Your version of Terraform is out of date! The latest version
+is 1.0.1. You can update by downloading from https://www.terraform.io/downloads.html
+```
+
+#### Set up the AWS CLI v2
+
+If you haven't already, set up the AWS CLI v2. The steps assume you already have an AWS account. 
+
+Follow the instructions in [Installing, updating, and uninstalling the AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+Next, FYI, refer to [Installing aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html). As explained on that page, 
+if you're running the AWS CLI version 1.16.156 or later, you don't need to install the authenticator.
+
+#### Install kubectl
+
+If you haven't already, install `kubectl`. You'll use `kubectl` commands to interact with your K8ssandra resources. 
+
+One option to get `kubectl` is described in this AWS topic, [Installing kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html). See the OS-specific examples. Here's an example on Linux and the 1.17 Kubernetes version:
+
+```bash
+curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.17.12/2020-11-02/bin/linux/amd64/kubectl
+```
+
+Verify the `kubectl` install:
+
+```bash
+kubectl version --short --client
+```
+
+**Output**:
+
+```bash
+Client Version: v1.17.12
+```
+
+#### Install helm v3
+
+If you haven't already, install Helm v3. See this EKS topic, [Using Helm with Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/helm.html). Note the  prerequisite: before you can install Helm charts on your Amazon EKS cluster, you must configure `kubectl` to work for Amazon EKS. If you have not already done this, see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+
+Once you've completed the prerequisites, see the [Helm install] steps for your OS. Here's a Linux example:
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 > get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+#### Install Python v3
+
+If you haven't already, install Python version 3.x for your OS. See the [python downloads](https://www.python.org/downloads/) page.
+
+
```

Comment:
Do we really need to cover installation of the various tools? That's the whole purpose of the links in the table.

Reply:
Follow-up, are we going to cover installation of tools in the other topics as well?

Reply:
My recommendation is we briefly cover these prereq install steps for newbie users along with links to related cloud docs. Yes, doing the same in our cloud install topics.